app/src/main/java/com/bogdan/codeforceswatcher/features/actions/ActionsAdapter.kt

```diff
@@ -115,24 +132,77 @@ class ActionsAdapter(
         }
     }
 
+    private fun bindFeedbackItem(viewHolder: FeedbackItemViewHolder, feedbackItem: ActionItem.FeedbackItem) = with(feedbackItem) {
```

Comment:
Break lines by arguments here for better readability. Also in AS we have auto-formatting: Cmd + Shift + L, which ensures consistent code style across different developers.